release/preview/alpine38/docker/Dockerfile

```
@@ -42,7 +42,8 @@ ENV PS_INSTALL_FOLDER=/opt/microsoft/powershell/$PS_INSTALL_VERSION \
     LC_ALL=en_US.UTF-8 \
     LANG=en_US.UTF-8 \
     # set a fixed location for the Module analysis cache
-    PSModuleAnalysisCachePath=/var/cache/microsoft/powershell/PSModuleAnalysisCache/ModuleAnalysisCache
+    PSModuleAnalysisCachePath=/var/cache/microsoft/powershell/PSModuleAnalysisCache/ModuleAnalysisCache \
```

Doesn't this need `ENV`?
```suggestion
    ENV PSModuleAnalysisCachePath=/var/cache/microsoft/powershell/PSModuleAnalysisCache/ModuleAnalysisCache \
```

---

No, it's already part of an `ENV` and it's better not to create a new layer.    It creates a smaller image.

---

Ah, didn't see the ENV starter.  My bad.

---

Comments are ignored with regards to line continuation in a Dockerfile